build/Build.Steps.Windows.cs

```diff
@@ -106,28 +106,37 @@ partial class Build
         {
             var aspNetProject = Solution.GetProjectByName(Projects.Tests.Applications.AspNet);
 
-            MSBuild(x => x
-                .SetConfiguration(BuildConfiguration)
-                .SetTargetPlatform(Platform)
-                .SetProperty("DeployOnBuild", true)
-                .SetMaxCpuCount(null)
-                    .SetProperty("PublishProfile", aspNetProject.Directory / "Properties" / "PublishProfiles" / $"FolderProfile.{BuildConfiguration}.pubxml")
-                    .SetTargetPath(aspNetProject));
-
             var localCopyTracerHome = aspNetProject.Directory / "bin" / "tracer-home";
             CopyDirectoryRecursively(TracerHomeDirectory, localCopyTracerHome);
```

Comment:
This is needed or not? It seems strange that we do not need it for `BuildDockerImage(wcfProject);` and we do need it for `BuildDockerImage(aspNetProject);`.

Reply:
It is needed for `AspNet`, because in order for `AspNet` autoregistration to work, `OpenTelemetry.AutoInstrumentation` needs to be installed in the GAC.
It will be needed for `WCF` once we have similar autoregistration there.

Reply:
You can consider adding a comment in code explaining why it is needed.